kview/src/main/java/dev/ikm/komet/kview/controls/skin/KLComponentControlSkin.java

```diff
@@ -211,7 +211,7 @@ private void setupDragNDrop() {
                             int nid = EntityService.get().nidForPublicId(dropInfo.publicId());
                             EntityProxy entity = EntityProxy.make(nid);
                             if (!(control.getParent() instanceof KLComponentSetControl componentSetControl) ||
-                                    !componentSetControl.getEntitiesList().contains(entity)) {
+                                    !componentSetControl.getEntitiesSet().contains(entity)) {
```

Comment:
I'm not sure we should be changing `KLComponentControlSkin.java` because it belongs to the `KLComponentControl`. Controls reference a Skin object.